### ISSUE TITLE
@change causes ember-template-lint to have coniptions 

### ISSUE THREAD

**Issue by NullVoxPopuli** [ember-a11y/ember-select-light#37]
**Describe the bug**

```
  5:2  error  Event handler methods like `change` should not be passed in as a component arguments  no-passed-in-event-handlers
```

![image](https://user-images.githubusercontent.com/199018/119820319-56400800-bebf-11eb-89c8-8f7954c56446.png)

Latest ember-template-lint at the time: 3.4.2

**To Reproduce**

Use the `@change` arg

**Expected behavior**

I don't get yelled at

**Comment by hergaiety** [comment]
Ah interesting, thanks for the logged issue! I'll try to look into it this week, seems straightforward to resolve.

**Comment by hergaiety** [comment]
@NullVoxPopuli thanks again for logging this issue. Just published v2.0.5 which wraps a deprecation warning around `@change` and offers an `@onChange` in its place.
